packages/litexa-html/README.md

```diff
@@ -44,7 +44,119 @@ project_dir
         └── html
 ```
 
-## Adding 'mark' SSML Tags
+## Alexa Web API Interface
+
+Integrating your skill with the Alexa Web API will allow it to both send and receive data
+from your web app.
```

Comment:
Would also mention the fact that it launches the web app.

Reply:
Do you need to specify a URL for the web app somewhere, or does it try to automatically launch something?

Reply:
looks like it's the start directive - unless there's an additional way it does that too.